superset-frontend/src/features/tags/tags.ts

```diff
@@ -89,11 +94,7 @@ export function fetchTags(
     endpoint: `/api/v1/${objectType}/${objectId}`,
   })
     .then(({ json }) =>
-      callback(
-        json.result.tags.filter(
-          (tag: Tag) => tag.name.indexOf(':') === -1 || includeTypes,
-        ),
-      ),
+      callback(json.result.tags.filter((tag: Tag) => tag.type === 1)),
```

Comment:
Using the `type` to filter so that tags with `:` can be used accordingly.

Reply:
@Vitor-Avila can you point to the enum object instead of the value? Something like TagType.custom instead of `===1`?